### PR TITLE
[IOTDB-2315] Modify some problems of metric

### PR DESCRIPTION
**Pull request by stormbroken**
1. Fix some error in metric doc model.
2. Fix some description of iotdb-metric.yml
3. Modify config of metric model into hot load way.
4. add `preDefinedMetrics` in iotdb-metric.yml

### CODE REVIEW

File: metrics/dropwizard-metrics/src/test/java/org/apache/iotdb/metrics/dropwizard/DropwizardMetricTestPlan.java [hunk 1]
```diff
@@ -52,7 +53,6 @@ private static void test(Integer metric, Integer tag) {
   }
 
   public static void main(String[] args) {
```
Comment (jixuan1989) [inline metrics/dropwizard-metrics/src/test/java/org/apache/iotdb/metrics/dropwizard/DropwizardMetricTestPlan.java:55]: where a main() method here?

Comment (stormbroken) [reply metrics/dropwizard-metrics/src/test/java/org/apache/iotdb/metrics/dropwizard/DropwizardMetricTestPlan.java:55]: This main method is only used for performance and memory test between dropwizard and micrometer, so whether should I need to remove it?

Comment (stormbroken) [reply metrics/dropwizard-metrics/src/test/java/org/apache/iotdb/metrics/dropwizard/DropwizardMetricTestPlan.java:55]: This main method is only used for performance and memory test between dropwizard and micrometer, so whether should I need to remove it?

**Review by xinzhongtianxia — APPROVED** [review]
LGTM

**Pull request merged**